implementation/domain/web/jbb-system-web/src/main/webapp/WEB-INF/templates/acp/system/sessions.html

```diff
@@ -0,0 +1,65 @@
+<html xmlns:th="http://www.thymeleaf.org"
+      xmlns="http://www.w3.org/1999/xhtml">
+<body>
+<div th:fragment="acpContent">
+    <div class="col-md-9">
+        <form th:action="@{/acp/system/sessions/setnewvalueofproperties}" th:object="${sessionSettingForm}" th:method="post">
+            <div th:if="${savecorrectly == true}"
+                 class="alert alert-dismissible alert-success">
+                <button type="button" class="close" data-dismiss="alert">×</button>
+                <label class="control-label">Settings saved correctly</label>
+            </div>
+
+            <div class="form-group">
+                <label class="control-label">Max inactive interval of session:</label>
+                <input type="text" class="form-control" id="intervalField"
+                       th:field="*{maxInactiveIntervalTime}">
+
+                <p th:if="${#fields.hasErrors('maxInactiveIntervalTime')}">Provided value can't be parsed to long!</p>
```

Comment:
"Provided value can't be parsed to long!" - it is not message for end user. Users don't know what is "long"